index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Metaverse Survey Form</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    
+
+    <section id="header">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12">
+                    <h2>Microverse Survey Form</h2>
+                    <p>Fill in every field correctly</p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section id="content">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-3"></div>
+                <div class="col-md-6">
+                    <div class="biodata">
+                        <form action="" method="post">
+                            <label for="name">Name</label>
+                            <input type="text" name="name" class="form-control" placeholder="Enter your name" required>
+                            <label for="email">Email</label>
+                            <input type="email" name="name" class="form-control" placeholder="Enter your email" required>
+                            <label for="name">Username</label>
+                            <input type="text" name="username" class="form-control" placeholder="Enter your username" required>
+                            <label for="name">Phone Number</label>
+                            <input type="tel" name="gsm" class="form-control" placeholder="Enter your Phone Number" required>
+                        </form>
+                    </div>
+                    <div class="survey">
+                            <label>How satisfied are you with our service</label>
+                            <textarea name="satisfy" id="" class="form-control" placeholder="" required></textarea>
+                            <label>Better ways to improved our service</label>
+                            <textarea name="satisfy" id="" class="form-control" required></textarea>
+                            <label for="">What feature/option could we add to make your experience better?</label>
+                            <textarea name="satisfy" id="" class="form-control" required></textarea>
+                            <label for="">Is there anything you'd like to add?</label>
+                            <textarea name="satisfy" id="" class="form-control" required></textarea>
+                            <button class="btn btn-warning btn-lg btn-block">Submit</button>
+                    </div>
+                </div>
+                <div class="col-md-3"></div>
+            </div>
+        </div>
+    </section>
+
+
```

Comment:
please use the Main tag to be the parent container in your codebase for more accessibility and SEO instead of just the section tag